src/makeRobotAccountant.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  function getSum(x) {
+    counter++;
+
+    return function(y) {
+      if (counter === 4) {
```

Comment:
I think check would be little better:
```suggestion
      if (counter > 3 && counter % 2 === 0) {
```